src/content/en/updates/2017/01/nic56.md

```diff
@@ -0,0 +1,140 @@
+  elements that scroll normally until sticking to the top of the viewport.
+* And [HTML5 by Default](#html5-by-default) is enabled for all users.
+
+I’m Pete LePage, let’s dive in and see what’s new for developers in Chrome 56! 
+
+<div class="clearfix"></div>
+
+## Web Bluetooth API {: #webbluetooth }
+
+Until now, the ability to interact with bluetooth devices has only been
+possible for native apps.
+
+With Chrome 56, your web apps can communicate with nearby Bluetooth Low Energy
+devices in a secure and private manner using the
+[Web Bluetooth API](//webbluetoothcg.github.io/web-bluetooth/). 
+
+The Web Bluetooth API uses the [GATT protocol](https://webbluetoothcg.github.io/web-bluetooth/),
+which enables your app to connect to devices such as light bulbs, toys,
+heart-rate monitors, LED displays and more with just a
+[few lines of JavaScript](https://googlechrome.github.io/samples/web-bluetooth/).
+Web Bluetooth can also be combined with Physical Web beacons to discover and control nearby devices.
+
+Francois has a great
+[article on Updates](/web/updates/2015/07/interact-with-ble-devices-on-the-web),
+be sure to check out some of neat [demos](https://github.com/WebBluetoothCG/demos)
+to go along with it. And be sure to check out the
+[Web Bluetooth Community](https://plus.google.com/communities/108953318610326025178).
+
+## CSS `position: sticky;` {: #position-sticky }
+
+Previously, building content headers that scrolled normally until sticking
+to the top of the viewport required listening to scroll events and
+switching an element’s position from relative to fixed at a specified threshold. 
+
+It was difficult to synchronize, and often results in small visual jumps.
+
+Chrome now supports CSS
+[`position: sticky;`](//developer.mozilla.org/en-US/docs/Web/CSS/position#Sticky_positioning),
+a new way to position elements. 
+
+An element that is position sticky, starts relative; but becomes fixed,
+after the element reaches a certain scroll position.
+
+Simply set position sticky, and set a threshold for it to become sticky.
+
+    h3 {
+      /* Element will be 'fixed' when it ... */
+      position: sticky;
+      /* ... is 10px from the top of the viewport */
+      top: 10px;
+    }
+
+Paul Kinlan has an [Updates](/web/updates/2016/12/position-sticky) post
+about it.
+
+## HTML5 By Default {: #html5-by-default }
+
+Last August, we announced that we’d be
+[moving to HTML5 By Default](//blog.google/products/chrome/flash-and-chrome/)
+to offer a safer, more power-efficient experience. This change disables Adobe Flash Player unless there’s a user indication that they want Flash content on specific sites, and eventually all websites will require the user’s permission to run Flash. 
+
+In Chrome 56,
+[HTML5 By Default has been enabled for all users](//blog.chromium.org/2016/12/roll-out-plan-for-html5-by-default.html ),
+which means they will be prompted to run Flash on a site-by-site basis
+for sites that they have never visited before. 
```

Comment:
This could be shortened to, '...they will be prompted to run Flash on sites they've never visited.'